applications/asset_tracker/src/main.c

```diff
@@ -1070,6 +1084,7 @@ static void modem_data_init(void)
 /**@brief Initializes the sensors that are used by the application. */
 static void sensors_init(void)
 {
+	accelerometer_init();
```

Comment:
we now have. orientation_detector_init(), flip_detection_init()  and accelerometer_init(). All in the same file and related to the orientation/flip detection. Could probably be put into the orientation detector?

Reply:
The idea is that the accelerometer will be used for other functionalities in the future, such as changing the data update frequency of the GPS data to the cloud based on movement. Since the accelerometer is not strictly used for flip detection would it therefore make sense to extract the accelerometer source code out of flip_detection_init(). 

If we get a data_threshold module in the future would accelerometer_init() probably be removed from main and handled within said module, at least with my current understanding. I have no opinion on whether orientation_detector_init() and flip_detection_init() could/should be merged, but maybe @jtguggedal or @joakimtoe have some input?